File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.py
S = list(input())                # 알파벳 입력받는 코드
c = 'abcdefghijklmnopqrstuvwxyz'        # 전체 알파벳 리스트를 c로 선언

for i in c:                      # 반복문
    if i in S:                    # S 안에 있는 알파벳의 인덱스를 저장해놨다가 인덱스 위치 출력
        print(S.index(i), end = ' ')    # 1칸 띄워야 되므로 end=' '는 필수!
    else:                          # 숫자가 없다면 -1 출력
        print(-1, end = ' ')

File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String str = br.readLine();

        int[] alphabet = new int[26];
        for (int i = 0; i < 26; i++) {
            alphabet[i] = -1;
        }

        for (int i = 0; i < str.length(); i++) {
            char c = str.charAt(i);
            int index = c - 'a';

            if (alphabet[index] == -1) {
                alphabet[index] = i;
            }
        }

        for (int i = 0; i < 26; i++) {
            System.out.print(alphabet[i] + " ");
        }

    }
}